operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/transformer/JobIdEncodingTransformer.java

```diff
@@ -26,18 +29,33 @@
     private static final String CLASSNAME = JobIdEncodingTransformer.class.getName();
     private static final Logger logger = Logger.getLogger(CLASSNAME);
 
-    // Encryption key used for JavaBatch Job ID
-    private static final SecretKeySpec BATCHJOBID_ENCRYPTION_KEY =  getJobIdEncryptionKey();
+    // Tenant Encryption key used for JavaBatch Job ID
+    private static ConcurrentMap<String, SecretKeySpec> KEY_MAP = new ConcurrentHashMap<>();
 
-    public JobIdEncodingTransformer() {
+    private static JobIdEncodingTransformer transformer = null;
+
+    private JobIdEncodingTransformer() {
         // No Operation
     }
 
+    /**
+     * get the instance
+     *
+     * @return
+     */
+    public static JobIdEncodingTransformer getInstance() {
+        if (transformer == null) {
+            transformer = new JobIdEncodingTransformer();
+        }
+        // final SecretKeySpec BATCHJOBID_ENCRYPTION_KEY = getJobIdEncryptionKey();
```

Comment:
missed this on my first pass through...would prefer commented out code to be deleted (if not, then a comment explaining what it is and why its commented out)

Reply:
good catch, not suppose to. be there now. I'll remove